docs/modules/ROOT/pages/how-tos/configuring/customizing-the-build.adoc

```diff
@@ -146,14 +256,15 @@ If you experience any issues with your customized pipeline, try the following so
 * If you believe that your desired parameter values are not being passed into the pipeline, make sure that your assignment of that value doesn't get overwritten later in the `.yaml` file.
 
 * If your new task is not appearing in the pipeline run, ensure the following:
-** You added it to the correct place in the `.yaml` files, so that it has the path `.spec.params` or `.pipelineSpec.params`.
+** You added it to the correct place in the `.yaml` files, so that it has the path `.spec.tasks` or `.pipelineSpec.tasks`.
 ** You specified a valid `runAfter` field, and that the task in that field completed successfully.
 
 * For problems with both parameters and tasks, make sure you committed your changes to the `.tekton` directory in the repository that {ProductName} references for the component.
 
-* If your build pipeline can no longer successfully run, your best option is to simply rebuild the `.tekton` directory:
-** Use the link:https://github.com/konflux-ci/testrepo/tree/main/.tekton[template `.tekton` directory] to rebuild your own `.tekton` directory.
-
 == Additional resources [[additional-resources]]
-* Visit the Tekton documentation that explains link:https://tekton.dev/docs/pipelines/taskruns/#specifying-the-target-task[how to use `taskSpec`] in a task declaration.
 
+* Tekton docs for link:https://tekton.dev/docs/pipelines/tasks/[Tasks], link:https://tekton.dev/docs/pipelines/pipelines/[Pipelines]
+  and link:https://tekton.dev/docs/pipelines/pipelineruns/[PipelineRuns]
```

Comment:
Nit: All three of these are missing in the glossary. Would you be able to add them?